modules/apps/dispatch/dispatch-test/src/testIntegration/java/com/liferay/dispatch/service/test/DispatchTriggerLocalServiceTest.java

```diff
@@ -298,6 +298,43 @@ user, _getRandomDispatchExecutorType(),
 		}
 	}
 
+	@Test
+	public void testScheduleWithLaterCronExpressionSet() throws Exception {
```

Comment:
This does not follow the pattern. The pattern must be test + <method we are testing> + other elements

For example: `testUpdateDispatchTriggerWithCronExpressions`

Reply:
Furthermore, I would include different test cases, like:

// startDate in the past and after the cron expression

// startDate in the past and before the cron expression

// startDate in the future and before the cron expression

// startDate in the future and after the cron expression

// startDate null

What do you think?